go.sum

```diff
@@ -1,15 +1,612 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
This file is suddenly a whole lot bigger - I don't know if this is expected

Reply:
Should be all good! Just more checksums for dependencies to support newer versions of Go I believe. I have run `go mod tidy`, so I think this is as slim as it's going to get with Go 1.13 as the minimum version.